22.RaytracedAO/main.cpp

```diff
@@ -754,7 +741,7 @@ int main(int argc, char** argv)
 	for(uint32_t s = 0u; s < globalMeta->m_global.m_sensors.size(); ++s)
 	{
 		if (s == sensorIndex)
-			continue; // skip it because we have already addes this one above
+			continue; // skip it because we have already added this one above
```

Comment:
don't do this maybe, and add them all

then fix up the iteration at line 777/764 to start at `sensorIndex`

Reply:
Yeah this would've been a great prediction, because I'm fixing a bug related to this.

The correct solution would be to add all and then save a "slice" into the sensor array for the non-interactive mode sensors.